sample/kotlin/src/main/kotlin/com/datadog/android/sample/NavActivity.kt

```diff
@@ -48,18 +57,30 @@ class NavActivity : AppCompatActivity() {
             R.id.navigation_view_pager -> {
                 startActivity(Intent(this, ViewPagerActivity::class.java))
             }
-            R.id.show_snack_bar->{
+            R.id.show_snack_bar -> {
                 Snackbar.make(
                     this.window.decorView.rootView,
                     "Demo message",
                     Snackbar.LENGTH_LONG
                 ).show()
-
             }
             else -> result = super.onOptionsItemSelected(item)
         }
         return result
     }
 
+    /**
+     * A native method that is implemented by the 'native-lib' native library,
+     * which is packaged with this application. It will throw a C++ exception
+     * and catch it in the signal handler which will be visible in the logs.
+     */
+    external fun crashAndGetExceptionMessage()
+
     // endregion
+
+    companion object {
+        init {
+            System.loadLibrary("native-lib");
```

Comment:
Are you loading the _same_ lib as the one provided in the SDK ? 
I think  a test would be more relevant to actually write a crash in a NDK lib that only exist in the sample.
Also you're loading the same lib twice, and although it should not have any issue according to the docs, I don't really like doing this.

Reply:
Same as above